evil-collection-free-keys.el

```diff
@@ -0,0 +1,42 @@
+;;; evil-collection-free-keys.el --- Evil bindings for free-keys -*- lexical-binding: t -*-
```

Comment:
Look into adding the free-keys entry to the evil-collection-mode-list here.

https://github.com/jojojames/evil-collection/blob/master/evil-collection.el

Reply:
Done.